README.md

```diff
@@ -16,31 +20,90 @@ here](https://github.com/IBM/cloudant-kafka-connector/blob/0.100.2-kafka-1.0.0/R
+4. Start Kafka.
+5. Start the connector (see below).
+
+Connector execution in Kafka is available through scripts in the Kafka install path:
+
+`$KAFKA_HOME/bin/connect-standalone.sh` or `$KAFKA_HOME/bin/connect-distributed.sh`
+
+Use the appropriate configuration files for standalone or distributed execution with Cloudant as source, as sink, or both.
+
+For example:
+- standalone execution with Cloudant as source:
+
+  ```
+  $KAFKA_HOME/bin/connect-standalone.sh \
+  $KAFKA_HOME/config/connect-standalone.properties \
+  $KAFKA_HOME/config/connect-cloudant-source.properties
+  ```
+
+- standalone execution with Cloudant as sink:
+
+  ```
+  $KAFKA_HOME/bin/connect-standalone.sh \
+  $KAFKA_HOME/config/connect-standalone.properties \
+  $KAFKA_HOME/config/connect-cloudant-sink.properties
+  ```
+
+- standalone execution with multiple configurations, one using Cloudant as source and one using Cloudant as sink:
+
+  ```
+  $KAFKA_HOME/bin/connect-standalone.sh \
+  $KAFKA_HOME/config/connect-standalone.properties \
+  $KAFKA_HOME/config/connect-cloudant-source.properties \
+  $KAFKA_HOME/config/connect-cloudant-sink.properties
+  ```
+
+Any number of connector configurations can be passed to the executing script.
 
-* Configuration
-* Usage
 
 ## Configuration
 
-The Cloudant Kafka connector is designed to be used with Kafka Connect API 3.x.
+As outlined above, the Cloudant Kafka connector can be configured in standalone or distributed mode according to 
+the [Kafka Connector documentation](https://kafka.apache.org/documentation.html#connect_configuring).
 
-The Cloudant Kafka connector can be configured in standalone or distributed mode according to 
-the [Kafka Connector documentation](https://kafka.apache.org/documentation.html#connect_configuring). At a minimum it is necessary to configure:
+The `connect-standalone` or `connect-distributed` configuration files contain default values which are necessary for all connectors, such as:
 
 1. `bootstrap.servers`
 2. If using a standalone worker `offset.storage.file.filename`.
+3. `offset.flush.interval.ms`
+
+### Connector configuration
+
+The `cloudant-source-example` and `cloudant-sink-example` properties files contain the minimum required to get started.
+For a full reference explaining all the connector options, see [here (source)](docs/configuration-reference-source.md) and
+[here (sink)](docs/configuration-reference-sink.md).
+
+#### Authentication
+
+In order to read from or write to Cloudant, some authentication properties need to be configured. These properties are common to both the source and sink connector, and are detailed in the configuration reference, linked above.
+
+A number of different authentication methods are supported. IAM authentication is the default and recommended method; see [locating your service credentials](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-locating-your-service-credentials) for details on how to find your IAM API key.
```

Comment:
I think any of `iam` `container` or `vpc` could be recommended as "IAM-based" so maybe a little re-wording here.

```suggestion
A number of different authentication methods are supported. IBM Cloud IAM-based authentication methods are recommended and the default is to use an IAM API key. See [locating your service credentials](https://cloud.ibm.com/docs/Cloudant?topic=Cloudant-locating-your-service-credentials) for details on how to find your IAM API key.
```